app/Filament/Subcontractor/Pages/ManageDocuments.php

```
@@ -56,34 +73,45 @@ public function form(Schema $schema): Schema
+                            ->label('Attestation d\'Assurance Décennale (en cours de validité)')
+                            ->acceptedFileTypes(['application/pdf'])
+                            ->disk('local')
+                            ->directory(fn () => 'third_parties/'.auth()->user()->contact->third_party_id.'/documents')
+                            ->getUploadedFileNameForStorageUsing(fn (TemporaryUploadedFile $file): string => 'decennale_insurance.pdf')
+                            ->downloadable(),
+                        FileUpload::make('kbis')
+                            ->label('Extrait Kbis (moins de 3 mois)')
+                            ->acceptedFileTypes(['application/pdf'])
+                            ->disk('local')
+                            ->directory(fn () => 'third_parties/'.auth()->user()->contact->third_party_id.'/documents')
+                            ->getUploadedFileNameForStorageUsing(fn (TemporaryUploadedFile $file): string => 'kbis.pdf')
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$file'. (undefined)

(UnusedFormalParameter)